File: 06/Java/TripleExpression/Multiply.java
package TripleExpression;

public class Multiply extends BinaryOperator implements TripleExpression {
    public Multiply(TripleExpression first, TripleExpression second) {
        super(first, second);
    }

    protected int apply(int a, int b) {
        return a * b;
    }
}

File: 06/CSharp/TripleExpressionApp/TripleExpressionApp/Multiply.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TripleExpressionApp
{
    public class Multiply : BinaryOperator
    {
        public Multiply(ITripleExpression first, ITripleExpression second)
        {
            fOp = first;
            sOp = second;
        }

        override protected double Apply(double a, double b)
        {
            return a * b;
        }
    }
}
